Gruntfile.js

```diff
@@ -253,6 +273,9 @@ module.exports = function(grunt) {
 
     require('jit-grunt')(grunt);
 
+    grunt.loadNpmTasks('grunt-selenium-webdriver');
```

Comment:
[jit-grunt](https://www.npmjs.com/package/jit-grunt) should load these automatically.